### ISSUE TITLE
findSetters is matching some wrong variables

### ISSUE THREAD

**Issue by callmeteus** [aaronbarker/css-variables-polyfill#18]
findSetters regex `(--[^:)]+:[\s]*[^;}]+)` is set to match multi-line characters, so, the following is being match as a variable name-value pair:

```
/* ----------------------------------------------------- */
@import url("https://fonts.googleapis.com/css?family=Roboto:300,400,500,700&display=swap");
```

I've fixed it using this regex instead of the original one: `(--[^:)\n\r]+:[\s]*[^;}]+)`

**Comment by aaronbarker** [comment]
Thanks for the report. I tweaked the regex a different way and made a test on regex101 to make sure this and other things worked. Will try to get it added soon.
https://regex101.com/r/kWwUmp/2

**Comment by aaronbarker** [comment]
And another tweak to handle dash/hyphens
https://regex101.com/r/kWwUmp/3

**Comment by aaronbarker** [comment]
This should be addressed in 1.2
